packages/blitz/src/cli/commands/db.ts

```diff
@@ -0,0 +1,90 @@
+import {CliCommand} from "../index"
+import arg from "arg"
+import {join} from "path"
+import {REGISTER_INSTANCE} from "ts-node"
+import chalk from "chalk"
+
+const args = arg(
+  {
+    // Types
+    "--help": Boolean,
+    "--env": String,
+    "--file": String,
+
+    // Aliases
+    "-h": "--help",
+    "-e": "--env",
+    "-f": "--file",
+  },
+  {
+    permissive: true,
+  },
+)
+
+const runSeed = async (seedBasePath: string) => {
+  if (!process[REGISTER_INSTANCE]) {
+    // During blitz interal dev, oclif automaticaly sets up ts-node so we have to check
+    require("ts-node").register({compilerOptions: {module: "commonjs"}})
+  }
+  require("tsconfig-paths/register")
```

Comment:
Why is it needed? We didn't have it in legacy blitz. I'm getting `MODULE_NOT_FOUND` errors with that line, and looks like it works fine without it